dataconnect/dataconnect/schema/schema.gql

```diff
@@ -0,0 +1,104 @@
+# Movies
+type Movie
+  @table(name: "Movies", singular: "movie", plural: "movies", key: ["id"]) {
```

Comment:
nit: Most of the time, default behavior is what they want. Verbose arguments scare people off. Let's omit them.

- `@table(...)`: change to `@table`. Only keep it for join tables.
- `@col`: Remove except for `Vector @col(size:768)`.
- `@ref`: Remove. Relation field has `@ref` implicitly.
- `@default(expr: "uuidV4()")` is default behavior of `UUID`, so can be removed. Need to keep `@default(expr: "request.time")` around because the default behavior is NULL.